hi-ml/src/health_ml/utils/data_augmentations.py

```diff
@@ -171,17 +171,34 @@ def __init__(self, kernel_size: int, p: float = 0.5, min: float = 0.1, max: floa
         self.kernel_size = kernel_size
         self.p = p
 
-    def __call__(self, sample: torch.Tensor) -> torch.Tensor:
+    @staticmethod
+    def apply_gaussian_blur(sample: torch.Tensor, kernel_size: int, p: float, min: float, max: float) -> torch.Tensor:
         prob = np.random.random_sample()
-
-        if prob < self.p:
-            sigma = (self.max - self.min) * np.random.random_sample() + self.min
-            sample = np.array(sample.squeeze())  # type: ignore
-            sample = cv2.GaussianBlur(sample, (self.kernel_size, self.kernel_size), sigma)
-            sample = torch.Tensor(sample).unsqueeze(0)
-
+        if prob < p:
+            sigma = (max - min) * np.random.random_sample() + min
+            sample = sample.permute([0, 2, 3, 1]).squeeze().numpy()   # only 3 channels, color channel last
+            sample = cv2.GaussianBlur(sample, (kernel_size, kernel_size), sigma)
+            sample = torch.Tensor(sample).unsqueeze(0).permute(0, 3, 1, 2)
```

Comment:
I know that you are just moving code around - but while you're touching it, can we maybe clean it up a bit?
- The check to apply could be written via a [binomial generator](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.binomial.html#numpy.random.Generator.binomial) as `if random.Generator.binomial(n=1, p=p):`
- It would be more readable to write `sigma = random.Generator.uniform(low=min,high=max)`
- Add documentation for the arguments of the function

Reply:
Thank you, I have made the suggested changes